docs/dev/new_check_howto.md

```diff
@@ -78,7 +78,7 @@ Answer the questions when prompted. Once completed succesfully, you will end up
     └── tox.ini
 ```
 
-## Write the check
+## Write the Integration check
```

Comment:
I think we should stick with one of these two terminologies:
1) "Check" (the original name, still broadly used among customers and in code references)
2) "Agent Integration" (the more "institutional" name that we introduce to consolidate nomenclature across the product) - in this case it could be just "Integration" since the context is clear.